[31mCMake Error at CMakeLists.txt:15 (find_package):
  find_package called with invalid argument "Python3"

[0m
-- Found ament_lint_auto: 0.12.10 (/opt/ros/humble/share/ament_lint_auto/cmake)
WARNING: Package name "UltrasonicData" does not follow the naming conventions. It should start with a lower case letter and only contain lower case letters, digits, underscores, and dashes.
-- Added test 'flake8' to check Python code syntax and style conventions
-- Added test 'lint_cmake' to check CMake code style
-- Added test 'pep257' to check Python code against some of the docstring style conventions in PEP 257
-- Added test 'xmllint' to check XML markup files
-- Configuring incomplete, errors occurred!
See also "/home/kalyani/ros2_arduino/build/UltrasonicData/CMakeFiles/CMakeOutput.log".
gmake: *** [Makefile:226: cmake_check_build_system] Error 1
